src/translations/fr.json

```diff
@@ -1540,9 +1540,10 @@
     "DiagramGridLayout": "Grille des diagrammes",
     "DiagramGridLayoutAndTree": "Grille des diagrammes et arbre décisionnel",
 
-    "CreateCouplingDevice": "Ajouter un couplage ou un omnibus",
+    "CREATE_COUPLING_DEVICE": "Ajouter un couplage ou un omnibus",
```

Comment:
what is your motivation of renaming CreateCouplingDevice to CREATE_COUPLING_DEVICE ? conflict?

Reply:
well to be more coherent with the menu

Reply:
![image](https://github.com/user-attachments/assets/9a8911b7-a518-48cc-9dbf-7655268fe0ad)